### PR TITLE
improved CollectionAffinityAttribute handling in the GroupBinding plugin

### PR DESCRIPTION
**Pull request by Fijo**

### CODE REVIEW

**Comment by Fijo** [comment]
This is the followup to #25 as I finally found the time when I also had the energy to sit down to implement it properly.

File: src/EcsRx.Plugins.GroupBinding/Systems/Handlers/GroupBindingSystemHandler.cs [hunk 1]
```diff
@@ -36,25 +37,26 @@ public GroupBindingSystemHandler(IObservableGroupManager observableGroupManager)
         public bool CanHandleSystem(ISystem system)
         { return true; }
 
-        public IGroup GetGroupAttributeIfAvailable(ISystem system, MemberInfo member)
+        public Tuple<IGroup, int[]> GetGroupAndAffinityFromAttributeIfAvailable(ISystem system, MemberInfo member)
```
Comment (grofit) [inline src/EcsRx.Plugins.GroupBinding/Systems/Handlers/GroupBindingSystemHandler.cs]: Could we use a named tuple here so its easier for consumer to see whats happening?
https://stackoverflow.com/questions/51670844/how-to-return-named-tuples-in-c

**Review by grofit — COMMENTED** [review]
Only one thing around changing generic tuple to named ones if possible, rest of it looks great thanks.

File: src/EcsRx.Plugins.GroupBinding/Systems/Handlers/GroupBindingSystemHandler.cs (hunk 1 above)
Comment (Fijo) [reply src/EcsRx.Plugins.GroupBinding/Systems/Handlers/GroupBindingSystemHandler.cs]: I didn't want to add a reference to System.ValueTuple just for that.
I made a dedicated class and used that instead.

**Pull request merged**